dependent_type_theory.md

```diff
@@ -355,11 +352,11 @@ You can also pass types as parameters:
 ```lean
 #check fun (α β γ : Type) (g : β → γ) (f : α → β) (x : α) => g (f x)
 ```
-The last expression, for example, denotes the function that takes
+This last expression, for example, denotes the function that takes
```

Comment:
The original text seems fine to me

Reply:
"This" instead of "the" makes it clearer that we are talking about the entire expression, not some subterm of it (perhaps even the subterm that comes last in the longer expression!). Perhaps a better fix would be, "The above expression, for example, ... "